client/my-sites/customer-home/cards/education/quick-start-video/index.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { Card } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import InlineSupportLink from 'components/inline-support-link';
+import { localizeUrl } from 'lib/i18n-utils';
+import { bumpStat, composeAnalytics, recordTracksEvent } from 'state/analytics/actions';
+import { FEATURE_QUICK_START_VIDEO } from 'my-sites/customer-home/cards/constants';
+import MaterialIcon from 'components/material-icon';
+
+/**
+ * Style dependencies
+ */
+import quickStartVideoImage from 'assets/images/customer-home/quick-start-video-ss.png';
+import './style.scss';
+
+export const QuickStartVideo = ( { trackQuickStartImpression } ) => {
+	const translate = useTranslate();
+
+	useEffect( () => {
+		trackQuickStartImpression();
+	}, [ trackQuickStartImpression ] );
+
+	return (
+		<div className="quick-start-video">
+			<h2 className="quick-start-video__heading customer-home__section-heading">
+				{ translate( 'Watch this video to get started' ) }
+			</h2>
+			<Card>
+				<div class="quick-start-video__content">
+					<div className="quick-start-video__content-wrapper">
+						<h3>{ translate( 'Video: Getting started on WordPress.com' ) }</h3>
+						<p className="quick-start-video__content-description customer-home__card-subheader">
+							{ translate(
+								'This video tour of WordPress.com will show you everything you need to know to start building your website.'
+							) }
+						</p>
+						<div className="quick-start-video__content-links">
+							<div className="quick-start-video__content-link">
+								<MaterialIcon icon="play_circle_outline" />
+								<InlineSupportLink
+									supportPostId={ 158974 }
+									supportLink={ localizeUrl(
+										'https://wordpress.com/support/getting-started-with-wordpress-com/#video-getting-started-with-word-press-com'
+									) }
+									showIcon={ false }
+									showText={ true }
+									tracksEvent="calypso_customer_home_education"
+									statsGroup="calypso_customer_home"
+									tracksOptions={ {
+										card_name: 'QuickStartVideo',
```

Comment:
We use the card id defined in the constants file in other cards ([example](https://github.com/Automattic/wp-calypso/blob/8df21cbe356e62ae5a25789c73e42d2e2cbc1f79/client/my-sites/customer-home/cards/education/free-photo-library/index.jsx#L37)), so I think it'd be good to keep doing it here:

```suggestion
										card_name: FEATURE_QUICK_START_VIDEO,
```